kube/stateful_set.go

```diff
@@ -52,7 +52,7 @@ func NewStatefulSet(role *model.Role, settings *ExportSettings) (*v1beta1.Statef
 }
 
 // getVolumeClaims returns the list of persistent volume claims from a role
-func getVolumeClaims(role *model.Role) []v1.PersistentVolumeClaim {
+func getVolumeClaims(role *model.Role, createHealmChart bool) []v1.PersistentVolumeClaim {
```

Comment:
Not a fan of bool arguments (hard to figure out what they are).

Random thought:
```golang
type KubeGeneratorType bool
const (
	KubeGeneratorRaw false
	KubeGeneratorHelm true
)
```

… okay that probably doesn't fit in this PR does it. Let's leave it alone for now and we can change later.